vertical-pod-autoscaler/enhancements/5030-vpa-manager/README.md

```diff
@@ -0,0 +1,338 @@
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+as worklods like deployments,statefulsets,etc. numbers growing, managing vpas become more and more complicated, because we need to manually create vpa for each of them. in more common scenarios, we just need to to create vpa for majority of workloads, and let vpa to extend or shrink workloads resources quietly. with vpa manager, we can just create or delete vpa in the scope of namespace, vpa manager will automaticlly create or delete for all workloads in the namespace.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+we want to make vpa more easier and automatically to use.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+- use labels autoscaling/vpa=open or autoscaling/vpa=open to create or delete vpa or set some parameters for workloads in namespaces
+- use labels vpa/update-mode=initial or vpa/update-mode=auto to set pod update policy
+- use annotations in workloads to make it exceptional in namespaces, for example, when we want to create vpa for all workloads except deployment/test, we can annotate deployment/test to do that.
+- 
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+- vpa manager don't manage all vpas, vpa manager just manager common and majority of workloads
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+  when we want to create vpa for all workloads in namespace a, we just label namespace a with autoscaling/vpa=open, then vpa manager create vpa for every workload not annotated with autoscaling/vpa=close in namespace a, and later when workload are created in namespace a, corresponding vpa will be automaticlly created.
```

Comment:
In this story there is no `vpa/update-mode` label. What mode do VPA objects have?

Reply:
thre's a default update mode configured in vpa manager launch parameters